src/bundle.c

```diff
@@ -103,6 +109,110 @@ static gboolean unsquashfs(const gchar *bundlename, const gchar *contentdir, con
 	return res;
 }
 
+static gboolean casync_make_arch(const gchar *idxpath, const gchar *contentpath, const gchar *store, GError **error) {
+	GSubprocess *sproc = NULL;
+	GError *ierror = NULL;
+	gboolean res = FALSE;
+	GPtrArray *args = g_ptr_array_new_full(15, g_free);
+	GPtrArray *iargs = g_ptr_array_new_full(15, g_free);
+	const gchar *tmpdir = NULL;
+
+	tmpdir = g_dir_make_tmp("arch-XXXXXX", &ierror);
```

Comment:
Does this need to be configurable? Root filesystems could be some GBs.

Reply:
Might be useful, I fear we have other things we should make configurable, too. Thus I would like to postpone this.

Reply:
OK.